packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -484,9 +486,45 @@ export class AtomicSearchInterface {
     ];
   }
 
+  private isI18nLocaleAvailable(locale: string) {
+    return availableLocales.includes(locale.toLowerCase());
+  }
+
   private get i18nBackendOptions(): BackendOptions {
     return {
-      loadPath: `${getAssetPath(this.languageAssetsPath)}/{{lng}}.json`,
+      loadPath: `${getAssetPath(
+        this.languageAssetsPath
+      )}/{{lng}}.json?lng={{lng}}&ns={{ns}}`,
+      request: async (_options, url, _payload, callback) => {
+        try {
+          const [fetchUrl, searchParams] = url.split('?');
+          const urlParams = new URLSearchParams(searchParams);
+          const lng = urlParams.get('lng')!;
+          const ns = urlParams.get('ns')!;
+
+          if (!this.isI18nLocaleAvailable(lng)) {
+            throw new Error(`Unsupported locale "${lng}"`);
```

Comment:
I would mention in the message that this isn't a locale we support out of the box, and link instructions on how to add locales.

Reply:
This won't be output to the user, it's caught right away

We shouldn't output a msg to the user here either, if he sets `es-ES` it will fail a first time but then fetch `es` which is fine too

Reply:
i18n basically fails quietly for unsupported languages and defaults to English